src/components/navigation/common/navigation-action.scss

```diff
@@ -12,7 +12,7 @@
   --sbb-navigation-action-color: var(--sbb-color-cloud-default);
 }
 
-:host([active]) {
+:host(:is([data-action-active], .sbb-active)) {
```

Comment:
Does the inclusion of `.sbb-active` not cause the possibility of multiple actions being visually active?